### PR TITLE
LOG-412. Update manifest for rh-certified operator pipeline

### PR DESCRIPTION
**Pull request by jcantrill**
Update manifest to support rh-certified operator pipeline: 

https://jira.coreos.com/browse/LOG-414

### CODE REVIEW

**Comment by ewolinetz** [comment]
other than image failures :+1:

File: Dockerfile [hunk 1]
```diff
@@ -4,13 +4,15 @@ COPY . .
...
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+ARG CSV=4.1
 
 ENV ALERTS_FILE_PATH="/etc/elasticsearch-operator/files/prometheus_alerts.yml"
 ENV RULES_FILE_PATH="/etc/elasticsearch-operator/files/prometheus_rules.yml"
 
 COPY --from=builder /go/src/github.com/openshift/elasticsearch-operator/_output/bin/elasticsearch-operator /usr/bin/
 COPY files/ /etc/elasticsearch-operator/files/
-ADD controller-manifests /manifests
+COPY /manifests/elasticsearch-operator.package.yaml /manifests
```
Comment (richm) [inline Dockerfile]: I think the source of the copy should not be an absolute path - try `manifests/` instead of `/manifests`

File: Dockerfile [hunk 2]
```diff
@@ -4,13 +4,15 @@ COPY . .
...
+ARG CSV=4.1
 
 ENV ALERTS_FILE_PATH="/etc/elasticsearch-operator/files/prometheus_alerts.yml"
 ENV RULES_FILE_PATH="/etc/elasticsearch-operator/files/prometheus_rules.yml"
 
 COPY --from=builder /go/src/github.com/openshift/elasticsearch-operator/_output/bin/elasticsearch-operator /usr/bin/
-COPY files/ /etc/elasticsearch-operator/files/
-ADD controller-manifests /manifests
+COPY --from=builder files/ /etc/elasticsearch-operator/files/
+COPY --from=builder manifests/elasticsearch-operator.package.yaml /manifests
```
Comment (bparees) [inline Dockerfile]: isn't this creating a file named "/manifests"?  hence why your next command fails because /manifests isn't a dir, it's a yaml file.

Comment (bparees) [reply Dockerfile]: i dont know why you wouldn't hit the same issue in https://github.com/openshift/cluster-logging-operator/pull/163/files#diff-3254677a7917c6c01f55212f86c57fbfR19

Comment (bparees) [reply Dockerfile]: (put a trailing / on it in both PRs to enforce it as a dir)

**Pull request merged**